history/management/commands/run_bot.py

```diff
@@ -155,26 +160,54 @@ def version(message):
                 "More info @ https://github.com/owocki/slack_gamebot " 
             message.reply(help_message)
 
+        def get_active_season(gamename,seasoned):
+            range_start_date = default_start
+
+            try:
+                active_season = Season.objects.get(gamename=gamename,active=True)
+            except Exception as e:
+                active_season = Season.objects.create(gamename=gamename,start_on = range_start_date,active = True)
+                active_season.save()
+
+            if seasoned:
+                range_start_date = active_season.start_on
+            else:
+                active_season = None
+
+            return active_season, range_start_date
+
+        @listen_to('^gamebot alltime leaderboard (.*)',re.IGNORECASE)
+        @listen_to('^gb alltime leaderboard (.*)', re.IGNORECASE)
+        @listen_to('^alltime leaderboard (.*)',re.IGNORECASE)
```

Comment:
I do think we should move towards the `gamebot` or `gb` prefix for all commands eventually. It's a habit-breaker for the office, but I think it makes more sense for gamebot in a general slackbot sense.

Neither here nor there for this PR, though.